app/features/history/card/EditHistorySetFormActions.js

```diff
@@ -10,41 +10,71 @@ import {
 } from 'app/ActionTypes';
 import * as Analytics from 'app/utility/Analytics';
 import * as SetsActionCreators from 'app/redux/shared_actions/SetsActionCreators';
+import * as SetsSelectors from 'app/redux/selectors/SetsSelectors';
+import * as DurationsSelectors from 'app/redux/selectors/DurationsSelectors';
+import * as DurationCalculator from 'app/utility/transforms/DurationCalculator';
```

Comment:
Move the duration calculation to DurationsSelectors so you don't have to use DurationCalculator here

Reply:
Done